### PR TITLE
CZ translations -radio

### PR DESCRIPTION
**Pull request by mhotar**

### CODE REVIEW

File: radio/util/translate.py [hunk 1]
```diff
@@ -18,7 +18,9 @@
                        ('\\207', u'š'),
                        ('\\207', u'Š'),
                        ('\\210', u'ú'),
+                       ('\\210', u'Ú'),
                        ('\\211', u'ů'),
+                       ('\\211', u'Ů'),
```
Comment (kilrah) [inline radio/util/translate.py:23]: These are not in the font!

Comment (mhotar) [reply radio/util/translate.py:23]: Right, but for monoscreens they are translated to lower case characters now. See the same code for both "ú" and "Ú"

Comment (kilrah) [reply radio/util/translate.py:23]: Ah yes, OK...

**Comment by bsongis** [comment]
Thanks!

**Pull request merged**